src/camera.cpp

```diff
@@ -40,6 +40,20 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #define WIELDMESH_OFFSET_X 55.0f
 #define WIELDMESH_OFFSET_Y -35.0f
 
+void Camera::onSettingsChange(const std::string &name){
+	if (name == "fov"){
+		m_cache_fov = g_settings->getFloat("fov");
+	}else if(name == "view_bobbing_amount"){
+		m_cache_view_bobbing_amount = g_settings->getFloat("view_bobbing_amount");
+	}else if(name == "fall_bobbing_amount"){
```

Comment:
I'm wondering why 'view/fall bobbing' settings are included here if they are not being changed in-game?

Reply:
I'm removing those in the next commit